JenkinsFile-Int

```diff
@@ -3,7 +3,7 @@ pipeline {
 		dockerfile {
 			filename 'Dockerfile'
 			label 'ec2-fleet'
-			additionalBuildArgs '--progress=plain'
+			additionalBuildArgs '--progress=plain --build-arg CODEARTIFACT_AUTH_TOKEN=${codeArtifactToken}'
```

Comment:
I think this needs to be CODEARTIFACT_AUTH_TOKEN not codeArtifactToken since you dont define the latter anywhere

Reply:
Fixed